modules/logging-release-notes-5-9-0.adoc

```diff
@@ -0,0 +1,79 @@
+
+[id="logging-release-notes-5-9-0-deprecation-notice"]
+== Deprecation notice
+
+* In {logging-uc} 5.9, Fluentd, and Kibana are deprecated and are planned to be removed in {logging-uc} 6.0, which is expected to be shipped alongside a future release of {product-title}. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements. The Vector-based collector provided by the {clo} and LokiStack provided by the {loki-op} are the preferred Operators for log collection and storage. We encourage all users to adopt the Vector and Loki log stack, as this will be the stack that will be enhanced going forward.
+
+* In {logging-uc} 5.9, the *Fields* option in output type the Splunk is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed.
+
+[id="logging-release-notes-5-9-0-enhancements"]
+== Enhancements
+
+[id="logging-release-notes-5-9-0-log-collection"]
+=== Log Collection
+* This enhancement adds the ability to refine the process of log collection by using a workload's metadata and to `drop` or `prune` logs based on their content. Additionally, it allows the collection of infrastructure logs, such as journal or container logs, and audit logs, such as `kube api` or `ovn` logs, to only collect individual sources. (link:https://issues.redhat.com/browse/LOG-2155[LOG-2155])
+
+* This enhancement introduces a new type of remote log receiver, the Syslog Receiver. You can configure it to expose a port over a network, allowing external systems to send Syslog logs using compatible tools such as Rsyslog. (link:https://issues.redhat.com/browse/LOG-3527[LOG-3527])
+
+* With this update, the `ClusterLogForwarder` API now supports log forwarding to Azure Monitor Logs, giving users better monitoring abilities. This feature helps users to maintain optimal system performance and streamline the log analysis processes in Azure Monitor, which speeds up issue resolution and improves operational efficiency. (link:https://issues.redhat.com/browse/LOG-4605[LOG-4605])
+
+* This enhancement improves collector resource utilization by deploying collectors as a deployment with two replicas. It happens when the only input source defined in the `ClusterLogForwarder` is a receiver input instead of using a daemon set on all nodes. Additionally, collectors deployed in this manner do not mount the host file system. To use this enhancement, you need to annotate the `ClusterLogForwarder` with `logging.openshift.io/dev-preview-enable-collector-as-deployment`. (link:https://issues.redhat.com/browse/LOG-4779[LOG-4779])
```

Comment:
```suggestion
* This enhancement improves collector resource utilization by deploying collectors as a deployment with two replicas. This occurs when the only input source defined in the `ClusterLogForwarder` custom resource (CR) is a receiver input instead of using a daemon set on all nodes. Additionally, collectors deployed in this manner do not mount the host file system. To use this enhancement, you need to annotate the `ClusterLogForwarder` CR with the `logging.openshift.io/dev-preview-enable-collector-as-deployment` annotation. (link:https://issues.redhat.com/browse/LOG-4779[LOG-4779])
```